docs/programming-languages.md

```diff
@@ -50,6 +50,17 @@ to learn.
 
 [Get started with JavaScript](https://github.com/wasabifan/ev3dev-lang-js){: .btn .btn-default }
 
+## Java
+<http://ev3dev-lang-java.github.io/>
+
+Java is a general-purpose computer programming language that is concurrent,
+
```

Comment:
The blank lines cause this to be 3 separate paragraphs.

https://github.com/ev3dev-lang-java/ev3dev.github.io/blob/c528b13073a600fa508eee160eef9434c7739931/docs/programming-languages.md#java

Please fix.